### PR TITLE
feat: add legislative tracker button

### LINKED ISSUE #866: Add link button to Legislative Tracker
Add a link button to the **Legislative Tracker** (https://www.notion.so/dcl-dao/Governance-Legislative-Tracker-Digest-BETA-7a9ef2f2f9d142d39cd7b33391ec97ae?pvs=4) within the **PROPOSALS** section.

### PR DESCRIPTION
**Pull request by andyesp**
Texts TBD with @ginocingolani 

**Screenshots**
image

**Related Issue**
Closes #866

### CODE REVIEW

**Review by 1emu — APPROVED** [review]

**Review by ncomerci — APPROVED** [review]

**Review by 1emu — APPROVED** [review]

**Comment by ginocingolani** [comment]
@andyesp 
Check every binding Governance decision approved by the community on this living document maintained by the Facilitation Squad
CTA: Legislative Tracker

File: src/intl/en.json [hunk 1]
```diff
@@ -1049,6 +1037,11 @@
         "starts": "Starts",
         "ends": "Ends",
         "ended": "Ended"
+      },
+      "legislative_tracker": {
+        "title": "How did we get here?",
+        "description": "Find all the binding outcomes from past DAO initiatives in this living document",
```
Comment (ginocingolani) [inline src/intl/en.json:1043]: title: How do we govern?
body: Check the latest binding Governance decisions the community approved on this living document maintained by the Facilitation Squad
CTA: Legislative Tracker

**Review by ginocingolani — CHANGES_REQUESTED** [review]
Copy proposed

**Review by ginocingolani — APPROVED** [review]
LGTM

**Pull request merged**